orchagent/portsorch.cpp

```diff
@@ -2232,6 +2232,12 @@ bool PortsOrch::addLag(string lag_alias)
     lag.m_members = set<string>();
     m_portList[lag_alias] = lag;
 
+    /* Add lag name map to counter table */
+    FieldValueTuple tuple(lag_alias, sai_serialize_object_id(lag_id));
+    vector<FieldValueTuple> fields;
+    fields.push_back(tuple);
+    m_counterTable->set("", fields);
```

Comment:
we need the remove part as well.

Reply:
The modification has been uploaded, please take a moment to review it. I want to continue uploading code here(sonic-swss), thanks.